extension/config/src/main/resources/conf/platform/exo-sample.properties

```diff
@@ -136,9 +136,9 @@
 # Configuration of the daily and weekly digests Cron Jobs
 # Use the Cron Expression syntax (http://www.quartz-scheduler.org/documentation/quartz-2.x/tutorials/tutorial-lesson-06)
 # Configuration of the daily digest email. By default it is sent at 11:30pm (23h30) every day.
-#exo.notification.NotificationDailyJob.expression=0 30 23 ? * *
+#exo.notification.NotificationDailyJob.expression=0 0 23 ? * *
```

Comment:
In the comment it is said 11:30 PM and in the cron expression it is for 11:00 PM, same for the Weekly diget @ line 141